components/automate-chef-io/content/docs/api-tokens.md

```diff
@@ -36,13 +36,13 @@ Admin tokens are tokens that are automatically added to the Administrator policy
 Admin tokens can only be created using the `chef-automate` command line.
 
 ```
-chef-automate iam token create <your-token-name> --admin
+chef-automate iam token create <your-token-id> --admin
```

Comment:
This command generates an id and name that match the value passed in. A conflict results in an error message about a conflicting id, not a conflicting name (we don't require uniqueness on name).